### ISSUE TITLE
goimports file stopped working in 0.9.250, was working before

### ISSUE THREAD

**Issue by Kane-Sendgrid** [go-lang-plugin-org/go-lang-idea-plugin#1538]
expected: when choosing goimports file from menu, should fix imports section
current: doesn't make any changes.

**Comment by dlsniper** [comment]
Hi @Kane-Sendgrid, can you please give me a bit more details? What happens when you select the goimports option from the menu? Is the option in the menu enabled in the first place? Does any window appear with any message? Can you see anything in the logs? Thank you.

**Comment by zolotov** [comment]
Also it would be nice to have source code and code that you're expecting to get after 'goimports' invocation

**Comment by anidotnet** [comment]
And also it looks like "Go fmt file" and "Go fmt project" are also not working in linux. Though in PATH every thing is set, still it is giving the below error while invoking from menu.

`exec: "gofmt": executable file not found in $PATH`

**Comment by dlsniper** [comment]
@anidotnet that's a different issue. For that you need to have the PATH env variable configured in ```.profile``` @zolotov please correct me if I'm wrong (I haven't tested it as I'm not using that part of the functionality)

**Comment by anidotnet** [comment]
@dlsniper, just an FYI, I can see proper command is getting invoked with proper path in Run window. The command itself works in external terminal but not in IDE. PATH is set in default profile zsh and bash. Let me know if it is specifically has to be .profile or not.

**Comment by dlsniper** [comment]
@anidotnet please open a different issue with this problem as it's not related to the one that @Kane-Sendgrid describes. We'll see if we can address it there. Thank you.

**Comment by Kane-Sendgrid** [comment]
@dlsniper menu item is not greyed, I can click it but nothing happens. I would expect it to run format the source and add missing imports/remove unused ones. No windows, error messages and nothing in the log, looks like it's just a noop. I've downgraded to 0.9.201 in the meantime, it works in that release.

**Comment by Kane-Sendgrid** [comment]
@zolotov here is an example of what I would expect
before goimports
```
package main

import "fmt"

func main() {
	println("test")
}
```
after:
```
package main

func main() {
	println("test")
}
```

**Comment by dlsniper** [comment]
I think I've managed to replicate this, I'll see if I can work on it to fix it before passing it to @zolotov. Thank you.

**Comment by dlsniper** [comment]
Fixed in #1540
